opendrift/export/io_netcdf.py

```diff
@@ -3,235 +3,82 @@
 from datetime import datetime, timedelta
 import logging; logging.captureWarnings(True); logger = logging.getLogger(__name__)
 import string
-from shutil import move
+import shutil
 import pandas as pd
 
 import numpy as np
+import xarray as xr
 from netCDF4 import Dataset, num2date, date2num
 from opendrift.models.basemodel import Mode
 
-# Module with functions to export/import trajectory data to/from netCDF file
-# Strives to be compliant with netCDF CF-convention on trajectories
-# https://cfconventions.org/Data/cf-conventions/cf-conventions-1.6/build/cf-conventions.html#idp8377728
-# https://geo-ide.noaa.gov/wiki/index.php?title=NODC_NetCDF_Trajectory_Template
-
-skip_parameters = ['ID']  # Do not write to file
+# Module with functions to export/import trajectory data to/from netCDF file.
+# Follows netCDF CF-convention on trajectories:
+# https://cfconventions.org/Data/cf-conventions/cf-conventions-1.12/cf-conventions.html#_multidimensional_array_representation_of_trajectories
 
+def datetime_from_datetime64(dt64):
```

Comment:
May get some help from pd.to_datetime here.

Reply:
One thing I have run into many times with xarray and netcdf + thredds is that xarray uses nano-seconds unless forced to do otherwise. As well as variable offset, which means that concatenating files using thredds aggregates do not work unless they have the same offset (reference for time). This is maybe not a problem here.

Reply:
ok, yes that sounds bad.
Presently there are some ad_hoc pd.to_datetime and methods like the above scattered around.

But we should probably be consistent on that all input datetimes are immediately converted to some internal standard (e.g. with pd.to_datetime).